pkg/apis/v1/labels.go

```diff
@@ -121,6 +121,7 @@ var (
 	AnnotationEC2NodeClassHash                = apis.Group + "/ec2nodeclass-hash"
 	AnnotationEC2NodeClassHashVersion         = apis.Group + "/ec2nodeclass-hash-version"
 	AnnotationInstanceTagged                  = apis.Group + "/tagged"
+	AnnotationAMIFamilyCompatibility          = apis.CompatibilityGroup + "/v1beta1-ami-family-conversion"
```

Comment:
Why is this different than the v1beta1 label?

Reply:
The annotations store two different pieces of information. The v1beta1 label stores the version, whereas this stores the family. @engedaam pointed out though that we don't need the v1beta1 label since we know the storage version is v1 and the version can't be lost.